cmake/OpenCVFindLibsGUI.cmake

```diff
@@ -38,7 +38,7 @@ endif()
 ocv_clear_vars(HAVE_GTK HAVE_GTK3 HAVE_GTHREAD HAVE_GTKGLEXT)
 if(WITH_GTK AND NOT HAVE_QT)
   if(NOT WITH_GTK_2_X)
-    ocv_check_modules(GTK3 gtk+-3.0)
+    ocv_check_modules(HAVE_GTK3 gtk+-3.0)
     if(HAVE_GTK3)
```

Comment:
This change is wrong.
Check `ocv_check_modules()` implementation - it sets corresponding `HAVE_*` vars.

P.S. Always clean CMake cache (clean build directory) during dependencies change.